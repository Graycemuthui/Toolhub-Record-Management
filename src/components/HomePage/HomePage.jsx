import * as React from "react";
import Button from "@mui/material/Button";
import Box from "@mui/material/Box";
import TextField from "@mui/material/TextField";

const HomePage = () => {
  return (
    <div className="home-page">
      <h1 className="heading">TooluHub Records</h1>
      <p className="description">
        TooluHub has records that all records that are edited and also updated.
        The Dashboard contains all the data and also the leaderboard contains
        the users and edited
      </p>
      <Box
        component="form"
        sx={{
          "& .MuiTextField-root": { m: 1, width: "25ch" },
        }}
        noValidate
        autoComplete="off"
      >
        <div>
          <TextField id="filled-basic" label="Edit tool" variant="filled" />
        </div>
        <Button variant="contained">SUBMIT</Button>
      </Box>
    </div>
  );
};

export default HomePage;
