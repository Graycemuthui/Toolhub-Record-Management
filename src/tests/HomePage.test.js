import React from "react";
import TestRenderer from "react-test-renderer";
import HomePage from "../../src/components/HomePage/HomePage";

it("HomePage render correctly", () => {
  const tree = TestRenderer.create(<HomePage />).toJSON();
  expect(tree).toMatchSnapshot();
});
