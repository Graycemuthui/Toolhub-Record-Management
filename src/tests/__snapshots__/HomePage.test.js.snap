// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage render correctly 1`] = `
<div
  className="home-page"
>
  <h1
    className="heading"
  >
    TooluHub Records
  </h1>
  <p
    className="description"
  >
    TooluHub has records that all records that are edited and also updated. The Dashboard contains all the data and also the leaderboard contains the users and edited
  </p>
  <form
    autoComplete="off"
    className="MuiBox-root css-13r4j3i"
    noValidate={true}
  >
    <div>
      <div
        className="MuiFormControl-root MuiTextField-root css-1u3bzj6-MuiFormControl-root-MuiTextField-root"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled MuiFormLabel-colorPrimary MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled css-e4w4as-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink={false}
          htmlFor="filled-basic"
          id="filled-basic-label"
        >
          Edit tool
        </label>
        <div
          className="MuiInputBase-root MuiFilledInput-root MuiFilledInput-underline MuiInputBase-colorPrimary MuiInputBase-formControl css-19mk8g1-MuiInputBase-root-MuiFilledInput-root"
          onClick={[Function]}
        >
          <input
            aria-invalid={false}
            autoFocus={false}
            className="MuiInputBase-input MuiFilledInput-input css-10botns-MuiInputBase-input-MuiFilledInput-input"
            disabled={false}
            id="filled-basic"
            onAnimationStart={[Function]}
            onBlur={[Function]}
            onChange={[Function]}
            onFocus={[Function]}
            required={false}
            type="text"
          />
        </div>
      </div>
    </div>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium MuiButton-root MuiButton-contained MuiButton-containedPrimary MuiButton-sizeMedium MuiButton-containedSizeMedium css-sghohy-MuiButtonBase-root-MuiButton-root"
      disabled={false}
      onBlur={[Function]}
      onContextMenu={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      SUBMIT
    </button>
  </form>
</div>
`;
